src/Microsoft.VisualStudio.Editors/PropPages/CodeAnalysisPropPage.Designer.vb

```diff
@@ -96,10 +96,10 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
             '
             'AnalysisLevelComboBox
             '
+            resources.ApplyResources(Me.AnalysisLevelComboBox, "AnalysisLevelComboBox")
```

Comment:
What's the purpose of changing the ordering here?

Reply:
This was changed automatically through the designer, can change it back and see if it's happy with it.

Reply:
After looking more, it's okay to leave this first as others have the same order on this file.